nbconvert/filters/tests/test_markdown.py

```diff
@@ -171,7 +190,8 @@ def test_markdown2html_math_paragraph(self):
         ]
 
         for case in cases:
-            self.assertIn(case, markdown2html(case))
+            s = markdown2html(case)
+            self.assertIn(case,self._unescape(s))
```

Comment:
Yes that was the original test before. Currently this transformation does not leave these samples unchanged, which is why the unescaping needs to happen. I get that to make the test pass as it is it's easier to not have to concern yourself with the escaped bits.

Also, if the idea of this test is that the math processing leaves markdown unaffected, we should probably include some `<`, `>` and `&` in the markdown section and not automatically unescape them (because the math processing is leaking out into the markdown processing). 

Also, this test should include a 

```latex

\begin{}
… 
\end{}

```

(i.e., including the new lines around the declaration) since, not being wrapped in any number of `$`, there is an absence of the easiest cues to delimiting math vs. markdown.